### DISCUSSION TITLE
Add table view for relational interfaces

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#8833] [Feature Requests]
before: 

![image](https://user-images.githubusercontent.com/8030134/137539774-b44b6d15-927d-458b-86e8-62d2615360ac.png)

after: 

![image](https://user-images.githubusercontent.com/8030134/137539989-237e1378-ee2b-4486-b55c-a16f366abbca.png)

**Comment by br41nslug** [comment]
This has been added
